src/spell/modules/spell/spell/src/mailbox.rs

```diff
@@ -0,0 +1,243 @@
+use eyre::WrapErr;
+use marine_rs_sdk::marine;
+use marine_sqlite_connector::State;
+
+use fluence_spell_dtos::value::{StringValue, UnitValue};
+
+use crate::auth::{is_by_creator, is_by_spell};
+use crate::kv::primitive::read_string;
+use crate::schema::db;
+
+
+#[marine]
+pub struct AllMailboxResult {
+    pub messages: Vec<String>,
+    pub success: bool,
+    pub error: String,
+}
+
+#[marine]
+pub fn store_mailbox(message: String) -> UnitValue {
```

Comment:
doc plz

Reply:
why name it `store_mailbox`? isn't it `mailbox_push`, a counterpart of `mailbox_pop`?